app/pages/home/index.jsx

```diff
@@ -91,6 +92,7 @@ class HomePage extends React.Component {
                                 <SidebarLink><a href="#section-about">About MHacks</a></SidebarLink>
```

Comment:
Also can you change the hrefs and ids to not have `section` in them (and match and thus work). I've been meaning to do it for a while but keep forgetting